### PR TITLE
PHPCS fixes

### LINKED ISSUE #33: Add PHPCS to Travis
... and likely once it's done we'll have some work to fix the module's coding style.

### PR DESCRIPTION
**Pull request by amitaibu**
#33

Next is adding Travis integration. This one is yours @AronNovak

### CODE REVIEW

File: logs_http.module [hunk 1]
```diff
@@ -15,20 +15,20 @@ use Drupal\logs_http\Logger\LogsHttpLogger;
...
  * handler exits.
  *
- * @param $exception
+ * @param \Throwable $exception
  *   The exception object that was thrown.
  *
  * @see _drupal_exception_handler()
  */
-function _logs_http_exception_handler($exception) {
+function _logs_http_exception_handler(\Throwable $exception) {
```
Comment (amitaibu) [inline logs_http.module:23]: Checking if tests will like the type hint. Also, we should probably move this to a helper class

Comment (alexpott) [reply logs_http.module:23]: See #36 - this code is not needed anymore :)

**Comment by AronNovak** [comment]
image
Travis + PHPCS in action

**Comment by AronNovak** [comment]
@amitaibu PHPCS is added, freezed at the current stable version.

**Comment by amitaibu** [comment]
Thanks @AronNovak

**Pull request merged**